Tests/TensorFlowTests/EpochsTests.swift

```diff
@@ -19,95 +19,156 @@ import XCTest
 var rng = ARC4RandomNumberGenerator(seed: [42])
 
 final class EpochsTests: XCTestCase {
-  func testBaseUse() {
-    // A mock item type that tracks if it was accessed or not
-    class AccessTracker {
+ 
+  // An element that keeps track of when it was first accessed.
```

Comment:
It doesn't, though, does it?  It is *used* to keep track of when other elements are accessed.

Reply:
Addressed.